tabular/src/autogluon/tabular/models/xgboost/hyperparameters/searchspaces.py

```diff
@@ -24,11 +24,11 @@ def get_base_searchspace():
         'learning_rate': Real(lower=5e-3, upper=0.2, default=0.1, log=True),
         'max_depth': Int(lower=3, upper=10, default=6),
         'min_child_weight': Int(lower=1, upper=5, default=1),
-        'gamma': Real(lower=0, upper=5, default=0.01),
-        'subsample': Real(lower=0.5, upper=1.0, default=1.0),
         'colsample_bytree': Real(lower=0.5, upper=1.0, default=1.0),
-        'reg_alpha': Real(lower=0.0, upper=10.0, default=0.0),
-        'reg_lambda': Real(lower=0.0, upper=10.0, default=1.0),
+        # 'gamma': Real(lower=0, upper=5, default=0),
+        # 'subsample': Real(lower=0.5, upper=1.0, default=1.0),
+        # 'reg_alpha': Real(lower=0.0, upper=10.0, default=0.0),
+        # 'reg_lambda': Real(lower=0.0, upper=10.0, default=1.0),
```

Comment:
I support these changes based on the performance data. I just want to double check that we prefer to leave these commented out rather than deleted (in which case we could still recover them from git history). If we leave them commented out, I like the idea of adding a inline comment about why they need to still be in the code while not being used. TIOLI

Reply:
+1

Reply:
Good point, added inline comment about why they are disabled.